src/components/ComputeModal.js

```diff
@@ -953,6 +1001,27 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
           }),
           h(DataprocDiskSelector, { value: computeConfig.workerDiskSize, onChange: updateComputeConfig('workerDiskSize') })
         ])
+      ]),
+      div({ style: { ...gridStyle, gridTemplateColumns: '0.25fr 8.5rem 1fr 5.5rem 1fr 5rem', marginTop: '1.5rem' } }, [
+        h(IdContainer, [
+          id => div({ style: { gridColumnEnd: 'span 3' } }, [
+            label({ htmlFor: id, style: computeStyles.label }, ['Location']),
+            h(InfoBox, { style: { marginLeft: '0.5rem' } }, [
+              'Cloud environments run in the same region as the workspace bucket and cannot be changed at this time.'
+            ]),
+            div({ style: { marginTop: '0.5rem' } }, [
+              h(Select, {
+                id,
+                isDisabled: true, // Initially releasing regionality supporting only us-central1 region
+                // isDisabled: computeExists, // Can't update location of existing environments
+                isSearchable: false,
+                value: computeConfig.computeRegion,
+                onChange: ({ value, locationType }) => updateComputeLocation(value, locationType),
+                options: _.flow(_.filter(l => l.value !== 'US'), _.sortBy('label'))(allRegions)
```

Comment:
One more magic string left:

```suggestion
                options: _.flow(_.filter(l => l.value !== defaultLocation), _.sortBy('label'))(allRegions)
```

Reply:
Done